site/content/docs/user/quick-start.md

```diff
@@ -135,13 +135,12 @@ kubectl apply -f my-manifest-using-my-image:unique-tag
 **Note**: The Kubernetes default pull policy is `IfNotPresent` unless
 the image tag is `:latest` in which case the default policy is `Always`.
 `IfNotPresent` causes the Kubelet to skip pulling an image if it already exists.
-If you want those images loaded into node to work as expected, please:
+If you want those images loaded into node to work as expected, please either:
 
-- don't use a `:latest` tag
-
-and / or:
-
-- specify `imagePullPolicy: IfNotPresent` or `imagePullPolicy: Never` on your container(s).
+- Leave `imagePullPolicy` unset so that it uses the default behavior
```

Comment:
The default image pull policy depends on the tag.

Reply:
Changed it to only say that you can only use the default if using unique tags.